testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1012,3 +1016,67 @@
     # ignored
   end
 end
+
+# Navigate to first system listed
+When(/^I select the first system in the list$/) do
+  step %(I follow the left menu "Systems > Overview")
+  within(:xpath, '//section') do
+    row = find(:xpath, "//div[@class='table-responsive']/table/tbody/tr[.//td]", match: :first)
+    # click first link in first row
+    within(row) do
+      first('a').click
+    end
+  end
+end
+
+# Change first system's property
+When(/^I have a system with "([^"]*)" as "([^"]*)" property$/) do |property_value, property_name|
+  # change system's property and clean the search index to make sure everything's correctly updated
+  steps %(
+    When I select the first system in the list
+    And I follow "Properties" in the content area
+    And I enter "#{property_value}" as "#{property_name}"
+    And I click on "Update Properties"
+    Then I should see a "System properties changed" text
+    And I clean the search index on the server
+    And I wait for "5" seconds
```

Comment:
Before anyone comments on this, I hate it more than you do but there didn't seem to be any other way to ensure the search server was working after the reindex if not with a forced wait :cry: open to ideas to improve this

Reply:
Sorry, but I will comment :trollface: 
If we use this step just as a precondition of something else, and not to really validate it.
I suggest that we explore the commands or XMLRPC call way, doing it with the WebUI looks like very much slower.

Reply:
About how to know when the re-index has been finished, I don't have any clue.
If you don't wait, what's the behavior? Maybe we can handle it differently by waiting until the element we want in the list is shown or not shown?

Reply:
If we don't wait, the search fails and there's a warning on the UI telling us it can't connect to the search server